### PR TITLE
Fixes #2609 - Add trophy icon next to the winning poll option

### LINKED ISSUE #2609: Make completed poll more clearly visible
with a 🏆 like on EW:
image

See the associated GH issue for context.

### PR DESCRIPTION
**Pull request by stefanceriu**
What the title says

### CODE REVIEW

File: ElementX/Sources/Screens/RoomScreen/View/Polls/PollOptionView.swift [hunk 1]
```diff
@@ -39,9 +39,19 @@ struct PollOptionView: View {
                         .frame(maxWidth: .infinity, alignment: .leading)
 
                     if showVotes {
-                        Text(L10n.commonPollVotesCount(pollOption.votes))
-                            .font(isFinalWinningOption ? .compound.bodySMSemibold : .compound.bodySM)
-                            .foregroundColor(isFinalWinningOption ? .compound.textPrimary : .compound.textSecondary)
+                        if isFinalWinningOption {
+                            HStack(spacing: 4) {
+                                Asset.Images.pollWinner.swiftUIImage
```
Comment (pixlwave) [inline ElementX/Sources/Screens/RoomScreen/View/Polls/PollOptionView.swift:44]: Should probably use `CompoundIcon(asset: Asset.Images.pollWinner)` here so it gets magic dynamic type support.

File: ElementX/Resources/Assets.xcassets/images/poll-winner.imageset/Contents.json
Comment (pixlwave) [inline ElementX/Resources/Assets.xcassets/images/poll-winner.imageset/Contents.json:1]: Can we
- enable template mode and colour it using compound
- enable preserves vector information (or whatever its called).

**Review by pixlwave — APPROVED** [review]
Nice, just a couple of small comments about using Compound more.

File: ElementX/Resources/Assets.xcassets/images/poll-winner.imageset/Contents.json
Comment (stefanceriu) [reply ElementX/Resources/Assets.xcassets/images/poll-winner.imageset/Contents.json:1]: Very good points, of course we can 😁

**Pull request merged**